extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -35,17 +35,13 @@
 }
 
 .i-amphtml-story-interactive-slider-bubble {
-  position: absolute !important;
+  position: relative;
   transform: translate(-50%, -50%) scale(0) !important;
-  width: 2.8em !important;
-  height: 2.8em !important;
-  top: 50% !important;
-  left: var(--percentage) !important;
   text-align: center !important;
   line-height: 2.9em !important;
   font-weight: 700 !important;
   color: white !important;
-  z-index: 1 !important;
+  transition: transform 0.5s var(--i-amphtml-interactive-ease-out-curve) !important;
```

Comment:
Please replace .5s with `var(--i-amphtml-interactive-animation-time)` here and all other instances in this file. 
it's .3s which is a more snappy duration and will match the animation time of the other interactive components.